internal/provider/resource_ably_rule_bodyguard_test.go

```
@@ -0,0 +1,250 @@
+					func(s *terraform.State) error {
+						rs, ok := s.RootModule().Resources["ably_rule_bodyguard.rule0"]
+						if !ok {
+							return fmt.Errorf("rule not in state")
+						}
+						url := fmt.Sprintf("%s/apps/%s/rules/%s",
+							os.Getenv("ABLY_URL"), rs.Primary.Attributes["app_id"], rs.Primary.ID)
+						httpResp, err := http.Get(url)
+						if err != nil {
+							return err
+						}
+						defer httpResp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `httpResp.Body.Close` is not checked

(errcheck)